src/build/app/mod.rs

```diff
@@ -106,6 +105,16 @@ impl<'b> App<'b> {
     pub fn get_name(&self) -> &str {
         &self.name
     }
+
+    /// Get the name of the binary
+    pub fn get_bin_name(&self) -> Option<&str> {
+        self.bin_name.as_deref()
+    }
+
+    /// Set binary name. Uses `&mut self` instead of `self`
+    pub fn set_bin_name<S: Into<String>>(&mut self, name: S) {
```

Comment:
Maybe name this `bin_name`? We should already have a `bin_name` builder somewhere around here.

Reply:
Yes, but `bin_name` moves `self instead of borrowing it mutably. We need it in here:

https://github.com/clap-rs/clap/blob/9060e3e56343c86f6a97c1e97aed83a9ac248e65/clap_generate/src/lib.rs#L176-L182

I'll adress this later, but I'm afraid not in this PR